src/figuresClasses.ts

```diff
@@ -1,19 +1,103 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape;
```

Comment:
we can skip `this.shape = 'triangle`
and write it here:
`shape: Shape = 'triangle'`


it's about all types of shapes

Reply:
done